### PR TITLE
Feature/page layout

### PR DESCRIPTION
**Pull request by jamarob**
Add grid layout

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -6,25 +6,37 @@ import {
...
+        </Switch>
+      </PageLayout>
     </Router>
   )
 }
 
-export default App
+const PageLayout = styled.div`
+  display: grid;
+  grid-template-rows: 48px auto;
```
Comment (arisslin) [inline src/App.js]: Replace   
  position: fixed;
  left: 0;
  right: 0;
  top: 0;
  bottom: 0;
  height: 100%;

with only height: 100vh;
Works also ;-)

File: src/components/NotesList.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ export default function NotesList({ notes }) {
 }
 
 const StyledNotesList = styled.section`
+  overflow-y: scroll;
   display: flex;
   flex-direction: column;
   gap: 24px;
```
Comment (arisslin) [inline src/components/NotesList.js]: gap doens't work with display flex onyly with grid
use margin-bottom: 24px in Note.js

File: src/GlobalStyle.js [hunk 3]
```diff
@@ -2,9 +2,14 @@ import { createGlobalStyle } from 'styled-components'
...
   * {
-    margin: 0;
-    padding: 0;
     box-sizing: border-box;
+    margin: 0;
+    padding:0;
+  }
+
+  body {
+    height: 100vh;
```
Comment (arisslin) [inline src/GlobalStyle.js]: height: 100vh;
overflow: hidden;
Both not needed beacause your app has all needed props

File: src/GlobalStyle.js [hunk 4]
```diff
@@ -2,9 +2,14 @@ import { createGlobalStyle } from 'styled-components'
 
 const GlobalStyle = createGlobalStyle`
   * {
-    margin: 0;
-    padding: 0;
     box-sizing: border-box;
+    margin: 0;
```
Comment (arisslin) [inline src/GlobalStyle.js]: Now all your html elements have padding 0 and margin 0 and you have to give them their default values back. Maybee it es wanted, but rethink it.

File: src/App.js (hunk 1 above)
Comment (jamarob) [reply src/App.js]: Much cleaner. Thanks!

File: src/GlobalStyle.js (hunk 4 above)
Comment (jamarob) [reply src/GlobalStyle.js]: Yeah, that was intentional.

File: src/GlobalStyle.js (hunk 3 above)
Comment (jamarob) [reply src/GlobalStyle.js]: Agreed. Don't know what I was thinking :)

File: src/components/NotesList.js (hunk 2 above)
Comment (jamarob) [reply src/components/NotesList.js]: Hm.. it works in firefox. I will keep it for now and investigate further.

**Pull request merged**